src/frontend/admin/playwright.config.ts

```diff
@@ -46,7 +46,7 @@ export default defineConfig({
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: "yarn start --port=8073",
```

Comment:
Instead, we should update the yarn start command into package.json.

```json
    "start": "npx serve out",
```
then you'll just have to replace the argument `--port` by `-p in playwright config
```suggestion
    command: "yarn start -p 8073",
```